pandas/io/packers.py

```diff
@@ -357,6 +357,7 @@ def encode(obj):
                     'klass': obj.__class__.__name__,
                     'axes': data.axes,
                     'blocks': [{'items': data.items.take(b.mgr_locs),
+                                'locs': b.mgr_locs.as_array,
```

Comment:
ok, you were looking at size. Well that is a trivial addition to size (this is one array per block, almost negligble). All that said I think we should remove this in the future (e.g. remove the forward compat, e.g. just use ``.items`` and just use ``.locs``). So pls create an issue and I'll it for the future to do this.